test/unit/features/debug.spec.ts

```diff
@@ -3,7 +3,7 @@ import { formatComponentName, warn } from 'core/util/debug'
 
 describe('Debug utilities', () => {
   it('properly format component names', () => {
-    const vm = new Vue()
+    const vm = new Vue('div')
```

Comment:
why passing `'div'`? it should work without it (javascript wise), if this is typescript error, please add a `@ts-expect-error` with a comment pls